repos/fdbt-site/src/layout/Navigation.tsx

```diff
@@ -1,9 +1,34 @@
 import React, { ReactElement } from 'react';
 
-const Navigation = (): ReactElement => (
+interface NavigationProps {
+    activePage: string;
```

Comment:
Just thinking we could work this out from the href of the window rather than passing it all the way through?